public/docs/ts/latest/guide/learning-angular.jade

```diff
@@ -0,0 +1,35 @@
+block includes
+  include ../_util-fns
+
+- var langName = current.path[1] == 'ts' ? 'TypeScript' : 'JavaScript'
+figure
+  img(src="/resources/images/devguide/intro/people.png" alt="Us" align="left" style="width:200px; margin-left:-40px;margin-right:10px" )
+:marked
+  Everyone is different. Everyone learns differently.

```

Comment:
Maybe drop the "Everyone is different", since "Everyone learns differently" seems to drive the point well enough. I'd merge this sentence with the next paragraph.